### ISSUE TITLE
interrupted binding request leaves config junk

### ISSUE THREAD

**Issue by postwait** [postwait/fq#9]
If a binding request is incomplete and the session is terminated, the queue and client and sockets remain.  This is likely true for other interrupted cmd messages.

**Comment by postwait** [comment]
```
-----------------  lwp# 15604 / thread# 15604  --------------------
 fffffd7ff37d742a nanosleep (fffffd7ffb931db0, 0)
 fffffd7ff37c12fc usleep (64) + 3c
 000000000040ae90 fqd_config_wait () + 40
 00000000004090af fqd_ccs_loop () + 3cf
 0000000000409372 fqd_command_and_control_server () + 192
 0000000000408584 conn_handler () + 204
 fffffd7ff37d0dba _thrp_setup (fffffd7ffefeda80) + 8a
 fffffd7ff37d10d0 _lwp_start ()
```

**Comment by postwait** [comment]
Fixed in fef4ffd0083ee51304bddc56843cf0427f5a0540
